doc/proposals/openshift-4.3/operator-testing-tool.md

```diff
@@ -0,0 +1,240 @@
+---
+title: operator-testing-tool
+authors:
+  - "@jmccormick2001"
+  - "@joelanford"
+reviewers:
+  - TBD
+  - "@joelanford"
+approvers:
+  - TBD
+creation-date: 2019-09-13
+last-updated: 2019-09-13
+status: provisional|implementable|implemented|deferred|rejected|withdrawn|replaced
+see-also:
+  - "/enhancements/this-other-neat-thing.md"  
+replaces:
+  - "/enhancements/that-less-than-great-idea.md"
+superseded-by:
+  - "/enhancements/our-past-effort.md"
+---
+
```

Comment:
It is not part of the template, wdyt about remove it? 
Also. note that the status are not the same suggested by @joelanford here: https://github.com/operator-framework/operator-sdk/pull/1929#discussion_r325229802

Reply:
@camilamacedo86 We're using [this template](https://github.com/openshift/enhancements/blob/master/enhancements/template.md) for enhancement proposal-style proposals.

If it makes sense, we could consider moving to that template for all future proposals.